profiles/notifications.py

```diff
@@ -10,3 +11,5 @@
     NotificationType.RELATIONSHIP_CONFIRMED.value,
     NotificationType.RELATIONSHIP_CONFIRMED.label,
 )
+
+dummy_context.context.update({"relationship": None})
```

Comment:
you can maybe make an in-memory `Relationship` object in the dummy context, so that you can get some of the fields in the preview.
```python
dummy_context.context.update({"relationship": Relationship(
    status=foo,
    other_field=bar
)})
```
at least that was the idea behind previews, not sure how it behaves in reality, as I haven't yet tested it 😅

Reply:
To create a dummy `Relationship`, I'd have to also create two dummy `Profiles` (which also requires a dummy `User`), and this didn't seem like a good place to do that.

Reply:
yeah, I was thinking, that it might be a problem. we can keept it the way it is now then 👍